justfile

```diff
@@ -72,7 +72,15 @@ format:
   cargo fmt --all
   nixpkgs-fmt $(echo **.nix)
 
-# start tmuxinator with a dev federation setup
+# start mprocs with a dev federation setup
+fed-shell:
+  ./scripts/fed-shell.sh
+
+# exit fed-shell session
+exit fed-shell:
```

Comment:
```suggestion
exit-fed-shell:
```

Reply:
Not sure we need that command, but if we do should be fixed